package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gusto/gusto-karma",
-  "version": "4.5.7",
+  "version": "4.6.0",
```

Comment:
is there a way to publish a `pre`/`beta` version? also why not 4.5.8?

Reply:
this isn't a bugfix. https://semver.org/

Reply:
we could do a beta version by adding  a `.beta` suffix. however, I'm not sure why we'd do that. I've already verified that this change works

Reply:
oh right. and okay great!